packages/loader/driver-utils/src/blobCacheStorageService.ts

```diff
@@ -17,12 +19,9 @@ export class BlobCacheStorageService extends DocumentStorageServiceProxy {
         super(internalStorageService);
     }
 
-    public async read(id: string): Promise<string> {
+    public async readBlob(id: string): Promise<ArrayBufferLike> {
         const blob = this.blobs.get(id);
-        if (blob !== undefined) {
-            return blob;
-        }
-
-        return this.internalStorageService.read(id);
+        assert(blob !== undefined, "blob is undefined");
```

Comment:
nit: Can you please add a more descriptive comment here. For example, why should blob not be undefined?